Update sdm660.dtsi

FammasMaz opened the pull request:
Testing EM busy costs based on caf stock em and linear regression as interpolation wrt frequency efficiency. Removed inefficient frequencies.

sohamxda7 commented:
> TTL Target is needed to fix TTL. This is very required option in some countries. Please enable it.
> If u enable TTL Target then u can easily fix TTL at any value u want by any program or script. For example by TTL fixer with root (https://github.com/ilya-fedin/TTLFixer/releases)

Thanks a ton :) :+1:  I will do it asap

MrArtemSid commented:
thx to u too)

FammasMaz committed:
Update sdm660.dtsi

Testing EM busy costs based on caf stock em and linear regression as interpolation wrt frequency efficiency. Removed inefficient frequencies.
